### PR TITLE
Front end/noty implementation

### PR DESCRIPTION
**Pull request by cchchoi1986**

### CODE REVIEW

File: app/assets/javascripts/angular/controllers/pages/recipe_finder_controller.js.erb [hunk 1]
```diff
@@ -1,4 +1,15 @@
-app.controller('RecipeFinderCtrl', ['$scope', '$http', '$routeParams', 'TagServices', 'UserServices', '$location', '$timeout', function($scope, $http, $routeParams, TagServices, UserServices, $location, $timeout){
+app.controller('RecipeFinderCtrl', ['$scope', '$http', '$routeParams', '$rootScope', 'TagServices', 'UserServices', '$location', '$timeout', 'Message', function($scope, $http, $routeParams, $rootScope, TagServices, UserServices, $location, $timeout, Message) {
+
+  $scope.visitMe = function () {
```
Comment (h8rry) [inline app/assets/javascripts/angular/controllers/pages/recipe_finder_controller.js.erb]: This is an empty function

File: app/assets/javascripts/angular/services/user_services.js [hunk 2]
```diff
@@ -39,5 +39,14 @@ app.factory('UserServices', ["$http", function($http){
   };
   user.resetSub();
 
+  // user.getCurrentUser = function() {
```
Comment (h8rry) [inline app/assets/javascripts/angular/services/user_services.js:42]: Can we delete it?

File: app/assets/javascripts/angular/controllers/user_account_controller.js.erb [hunk 3]
```diff
@@ -1,7 +1,21 @@
...
+    if (UserServices.status) {
+      $location.path("/myrecipes/"+UserServices.userID)
+    } else {
+      Message.sendNoty("error","Please log in to access");
+    }
+  }
 
   $scope.user = UserServices;
 
+// TODO: PUT ALL THAT IN THE USER SERVICE!
```
Comment (h8rry) [inline app/assets/javascripts/angular/controllers/user_account_controller.js.erb:18]: Yes

**Comment by h8rry** [comment]
The rest looks okay

File: app/assets/javascripts/angular/controllers/pages/recipe_finder_controller.js.erb (hunk 1 above)
Comment (cchchoi1986) [inline app/assets/javascripts/angular/controllers/pages/recipe_finder_controller.js.erb]: deleted

File: app/assets/javascripts/angular/services/user_services.js (hunk 2 above)
Comment (cchchoi1986) [inline app/assets/javascripts/angular/services/user_services.js:42]: deleted

File: app/assets/javascripts/angular/controllers/user_account_controller.js.erb (hunk 3 above)
Comment (cchchoi1986) [inline app/assets/javascripts/angular/controllers/user_account_controller.js.erb:18]: in good time sir

**Pull request merged**